example-deployment/k8s/reporting-server.yaml

```diff
@@ -0,0 +1,143 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: reporting-server-db
+  labels:
+    app: reporting-server
+    tier: db
+spec:
+  accessModes:
+    - ReadWriteOnce
+  volumeMode: Filesystem
+  resources:
+    requests:
+      storage: 1Gi
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: reporting-server-db
+  labels:
+    app: reporting-server
+    tier: db
+spec:
+  selector:
+    app: reporting-server
+    tier: db
+  ports:
+    - port: 5432
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: reporting-server-db
+  labels:
+    app: reporting-server
+    tier: db
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: reporting-server
+      tier: db
+  template:
+    metadata:
+      labels:
+        app: reporting-server
+        tier: db
+    spec:
+      containers:
+        - name: postgresql
+          image: postgres:13
+          env:
+            - name: POSTGRES_USER
+              value: reporting-server
+            - name: POSTGRES_PASSWORD
+              value: reporting-server
+          volumeMounts:
+            - mountPath: /var/lib/postgresql/data
+              name: reporting-server-db
+      volumes:
+        - name: reporting-server-db
+          persistentVolumeClaim:
+            claimName: reporting-server-db
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: reporting-server
+  labels:
+    app: reporting-server
+    tier: app
+spec:
+  type: LoadBalancer
+  selector:
+    app: reporting-server
```

Comment:
This could be copied from the buggy yaml in the other example k8s files. It needs to have the
`tier: app` selector to prevent conflicts.

Reply:
Oh! Thanks for this!